server/command.go

```diff
@@ -788,6 +791,34 @@ func executeInstanceInstallCloud(p *Plugin, c *plugin.Context, header *model.Com
 	})
 }
 
+func executeInstanceInstallCloudOAuth(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
+	authorized, err := authorizedSysAdmin(p, header.UserId)
+	if err != nil {
+		return p.responsef(header, "%v", err)
+	}
+	if !authorized {
+		return p.responsef(header, "`/jira install` can only be run by a system administrator.")
+	}
+	if len(args) != 1 {
+		return p.help(header)
+	}
+	jiraURL, instance, err := p.installCloudOAuthInstance(args[0])
```

Comment:
Can we put a new line before this line? It helps separate the error/short-circuit cases and success cases

Reply:
yep, I like the code to "breathe" too, but it's probably gonna change a lot.